posts/tests.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from categories.models import Category, MainCategory
+
+
+class CreatePostViewTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser",
+            password="testpassword123",
+        )
+
+        self.main_category = MainCategory.objects.create(
+            name="MainCategory",
+        )
+
+        self.category = Category.objects.create(
+            name="Test Category",
+            main_category=self.main_category,
+        )
+
+        self.create_view_url_get = reverse(
+            "posts:create", kwargs={"category_id": self.category.pk}
+        )
+        self.create_view_url_post = reverse(
+            "posts:create", kwargs={"category_id": self.category.pk}
+        )
```

Comment:
suppose first url should get post

Reply:
get - should render PostForm
post - create Post

Reply:
I see, it is valid but it is not clear that self.create_view_url_get and self.create_view_url_post have the same url. So better to have only one consistent
```suggestion
        self.create_view_url = reverse(
            "posts:create", kwargs={"category_id": self.category.pk}
        )
```

Reply:
Oh, I see it, but don't pay attention.